final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,181 @@
+import argparse
+import feedparser
+import logging
+import json
+import html
+import sys
+import datetime
+
+__version__ = '3.5'
+args = None
+limit = 3
+links = []
+print_setting = {
+    'feed': [
+        'title'
+    ],
+    'entries': [
+        'title',
+        'published',
+        'link',
+        'summary'
+    ]
+}
+
+
+def get_object_feed(url: str):
```

Comment:
Here and below:
It's a good practice to add type hits in your function signatures (e.g. `def get_object_feed(url: str) -> str:`)